src/backend/src/services/organizations.service.ts

```diff
@@ -53,4 +55,26 @@ export default class OrganizationsService {
 
     return newLinks;
   }
+
+  static async getAllUsefulLinks(submitter: User, organizationId: string) {
+    if (!(await userHasPermission(submitter.userId, organizationId, isAdmin)))
```

Comment:
do you need this here?

Reply:
I removed it in the other PR already but good catch